ghost/core/core/server/services/email-suppression-list/MailgunEmailSuppressionList.js

```diff
@@ -96,6 +96,11 @@ class MailgunEmailSuppressionList extends AbstractEmailSuppressionList {
 
     async init() {
         const handleEvent = reason => async (event) => {
+            if (reason === 'bounce') {
+                if (event.error.code < 500 || event.error.code > 599) {
```

Comment:
Do we definitely always have event.error.code?

Reply:
Ah okay, yeah we should deffo update the JSDoc to reflect reality ☺️